docs/source/contributing/community.md

```diff
@@ -0,0 +1,27 @@
+# Community communication channels
+
+We use different channels of communication for different purposes. Whichever one you use will depend on what kind of communication you want to engage in.
+
+## Discourse
+
+We use [Discourse](https://discourse.jupyter.org) for online discussions. Everyone in the Jupyter community is welcome to bring ideas and questions there.
```

Comment:
I believe it is a good idea to be explicit about the fact that the Discourse channel is where support questions should live

```suggestion
We use [Discourse](https://discourse.jupyter.org) for online discussions and support questions. Everyone in the Jupyter community is welcome to bring ideas and questions there.
```